mantidproject/mantid
# Indirect Diffraction - OSIRIS rebin

Added a rebin step to OSIRIS reduction to allow differently sized container and sample files.

**To test:**

Requires access to the ISIS data archive.
Interfaces>Indirect>Diffraction
Change instrument to `OSIRIS` and mode to `diffonly`
Input Files: `119977-119981`
Check "use container" and input `71986-71990`
Cal File: [osiris_041_RES10.zip](https://github.com/mantidproject/mantid/files/787862/osiris_041_RES10.zip)
Vanadium Runs: `119963-119967`
Press `Run`

Fixes #18929 .

_Release Notes have been updated_

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
